pyproject.toml

```diff
@@ -1,12 +1,28 @@
 [build-system]
 requires = [
-    "setuptools >= 36.4",
+    "setuptools >= 62.5",
     "wheel",
-    "setuptools_scm >= 2.1",
+    "setuptools_scm[toml] >= 6.2",
     "cython >= 0.28.1",
 ]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "uharfbuzz"
+description = "Streamlined Cython bindings for the HarfBuzz shaping engine"
+readme = "README.md"
+license = {file = "LICENSE"}
+requires-python = ">=3.5"
```

Comment:
Should this be set to `>= 3.6`?

Reply:
i guess, since we no longer build wheels for 3.5 so we are not testing it agaist that. It's EOL so we may as well explictly drop support

Reply:
Oh, actually. setupttols 62.5 requires >= 3.7. Don't mind if I do :)

Reply:
now.. these are the kind of breaking changes for change sake that I really dislike.

Reply:
hold on! python3.6 also is EOL. I got confused, sorry. 😅 
yes, that's fine then